flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1875,7 +1984,7 @@ class CreateFilesystemTests(
 class MountBlockDeviceInitTests(
     make_with_init_tests(
         MountBlockDevice,
-        dict(volume=_ARBITRARY_VOLUME, mountpoint=FilePath(b"/foo")),
```

Comment:
OMG. I think I might be responsible for the `make_with_init_tests` function, but it seems like overkill now.
Shall we delete these?
Or at least supply the two dictionaries as keyword arguments so that the reader doesn't have go digging into flocker.testtools.